openmetadata-ui/src/main/resources/ui/src/utils/EntityUtils.tsx

```diff
@@ -1491,6 +1496,21 @@ export const getBreadcrumbForContainer = (data: {
   ];
 };
 
+export const getBreadcrumbForTestCase = (entity: TestCase) => [
+  {
+    name: i18next.t('label.incident-manager'),
+    url: ROUTES.INCIDENT_MANAGER,
+  },
+
+  {
+    name: entity.name,
+    url: getEntityLinkFromType(
+      entity.fullyQualifiedName ?? '',
+      (entity as SourceType).entityType as EntityType
```

Comment:
Here we should use optional chaining as we ar type casting and `entityType` can be undefined.

```suggestion
      (entity as SourceType)?.entityType as EntityType
```